src/batch/src/executor/delete.rs

```diff
@@ -72,6 +81,23 @@ impl DeleteExecutor {
             },
             identity,
             returning,
+            txn_id,
+            write_handle: None,
+            finished: false,
+        }
+    }
+}
+
+impl Drop for DeleteExecutor {
+    fn drop(&mut self) {
+        // If this executor is not finished, try to rollback.
+        if !self.finished {
+            if let Some(write_handle) = self.write_handle.take() {
+                let txn_id = self.txn_id;
+                tokio::spawn(async move {
```

Comment:
This looks a little bit indeterministic to me. 😰 I guess it's better to synchronously send the `Rollback` message, but now it's bounded.

Reply:
Use an unbundled channel to send messages for DMLs.